test/verify/check-system-info

```diff
@@ -818,7 +818,8 @@ password=foobar
         wait(lambda: progressValue(1) > 75)
         m.execute("pkill -e -f [c]at.*urandom")
         # should go back to idle usage
-        wait(lambda: progressValue(1) < 20)
+        # HACK: work around pmie CPU usage https://bugzilla.redhat.com/show_bug.cgi?id=2140572
+        wait(lambda: progressValue(1) < 20, delay=200)
```

Comment:
Ugh, no -- that severely penalizes all other distros where this works. I think you mean `tries=200`.

Reply:
Woops, that was me thinking about raising the delay or tries...